src/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: $main-background-color;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__image {
+    object-position: center;
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__product-title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    height: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    font-weight: 500;
+    color: $bold-font-color;
+  }
+
+  &__product-code {
+    align-self: start;
+    margin-top: 4px;
+    margin-bottom: 0;
+    width: 98px;
+    height: 14px;
+    line-height: 14px;
+    color: $main-font-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    height: 16px;
+    margin-top: 16px;
+  }
+
+  &__price {
+    height: 18px;
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-name {
+      color: $main-font-color;
+      margin: 0;
+      width: 31px;
+      font-size: 12px;
+      line-height: 18px;
+      height: 18px;
+    }
+
+    &-value {
+      margin: 0;
+      line-height: 18px;
+      font-size: 16px;
+      font-weight: 700;
+      width: 50px;
+    }
+  }
+
+  &__button-buy {
+    height: 40px;
+    border: none;
+    background-color: $extra-color;
+    color: rgba(255, 255, 255, 1);
+    font-size: 14px;
+    font-weight: 700;
+    border-radius: 5px;
+    margin-top: 16px;
+    margin-bottom: 16px;
+
+    &:hover {
+      border: 1px solid $extra-color;
+      background-color: $main-background-color;
+      color: $extra-color;
```

Comment:
Ensure that the variable `$extra-color` is defined in your SCSS files. If it's not defined, it will cause a compilation error.